hera_pspec/utils.py

```diff
@@ -1120,4 +1120,54 @@ def get_reds(uvd, bl_error_tol=1.0, pick_data_ants=False, bl_len_range=(0, 1e4),
 
     return reds, lens, angs
 
+def pspecdata_time_difference(ds, time_diff):
+    """
+    Given a PSpecData object and a time difference, give the time difference PSpecData object.  
```

Comment:
This seems like it could be very useful. Is the idea to estimate the covariance from time difference? I'm all for it. I think expanding the description here would be helpful since I'm not 100% sure if that's what this function is supposed to do.